reference-architectures/deploy-arch-ibm-slz-vsi-extension.md

```diff
@@ -0,0 +1,105 @@
+---
+
+copyright:
+  years: 2023
+lastupdated: "2023-12-08"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Vincent Burckhardt"
+
+# The release that the reference architecture describes
+version: 5.1.0
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: true
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global
```

Comment:
@ocofaigh , @SirSpidey 
Will the vsi-extension show up as a variation in the VSI catalog, or would it be a whole new separate catalog item?
![Screenshot 2023-12-11 at 9 51 55 AM](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/assets/122446118/1c366382-10ec-4b89-b63f-76b6b6c9fb08)

If its a variation in vsi on vpc landing zone, should the deployment-url be the one for vsi?

Reply:
It will show as a new variation exactly where you have that red box

Reply:
So, the deployment url would be the same as the vsi-pattern right?

Reply:
@Aashiq-J @ocofaigh 

> It will show as a new variation exactly where you have that red box

Actually I think it shows up like this ([comment in main PR](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/610#issuecomment-1781351513)), with a drop-down that says `Extends <name of base DA>`. Look at the PowerVS on VPC DA to see a real example. 

So it appears in either the base vpc or base Red Hat DA, not the VSI one (because it's adding the VSI to the DA).

> So, the deployment url would be the same as the vsi-pattern right?

We have 2 deployment URLs (**VPC** landing zone and **Red Hat OpenShift Container Platform on VPC** landing zone). We could try putting in both and see what happens, or just pick the most common one (🤷 VPC?)

Reply:
from the ibm_catalog.json file: 
flavors.dependencies.flavors.name: `deploy-arch-ibm-slz-ocp` and `deploy-arch-ibm-slz-vpc`.

Reply:
yep I stand corrected - it shows like this:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/assets/29608224/d5592c77-7107-4c55-b619-06ea94f771a6)